gio/src/subclass/action_group.rs

```diff
@@ -115,7 +111,7 @@ pub trait ActionGroupImplExt: ObjectSubclass {
         action_name: &str,
         value: &Variant,
     );
-    fn parent_get_action_enabled(&self, action_group: &Self::Type, action_name: &str) -> bool;
+    fn parent_action_is_enabled(&self, action_group: &Self::Type, action_name: &str) -> bool;
```

Comment:
Interesting, some of the parent_get_x were renamed, but not all of them

Reply:
Yeah, those were manual changes, will check again.